src/router/httpd/visuals/sysctl.c

```diff
@@ -69,6 +69,7 @@ static void showsysctl(char *path, char *nvname, char *name, char *sysval, void
 	websWrite(wp, "<div class=\"label\">%s</div>\n", name);
 	websWrite(wp, "<input maxlength=\"100\" size=\"40\" name=\"%s\" value=\"%s\" />\n", nvname, sysval);
 	websWrite(wp, "</div>\n");
+	websWrite(wp, "<br />\n");
```

Comment:
Im hoping if I understand this right, that a break is inserted between fieldsets so its consistent with all other pages.

current is like no break

![Capture](https://user-images.githubusercontent.com/31389848/135702137-b45adc6c-5d9b-4e11-b288-5ac734902375.PNG)

desired as mock show below.


![Capture](https://user-images.githubusercontent.com/31389848/135702219-8cfd73a7-4f96-49e4-b867-70cfe7b66773.PNG)